lib/src/settings/distribution.dart

```diff
@@ -0,0 +1,21 @@
+class Distribution {
+  final double mean;
+  final double sigma;
+
+  Distribution({
+    required this.mean,
+    required this.sigma,
+  });
+
+  factory Distribution.fromMap(Map<String, Object?> map) {
+    return Distribution(
+      mean: map['mean'] as double,
+      sigma: map['sigma'] as double,
+    );
+  }
+
+  Map<String, Object?> toMap() => {
+        'mean': mean,
+        'sigma': sigma,
+      };
```

Comment:
since this is used in query parameters, it should automatically exclude null values (as per meilisearch conventions), the logic for this is already present in `Queryable` class, so it can be uesd.

Reply:
Distribution isn't used in the query parameters; it is a field of some embedders, and the fields in this class aren't actually nullable.